customer-commons/src/main/java/no/unit/nva/customer/model/channelclaim/ChannelClaimDto.java

```diff
@@ -0,0 +1,18 @@
+package no.unit.nva.customer.model.channelclaim;
+
+import com.fasterxml.jackson.annotation.JsonTypeInfo;
+import com.fasterxml.jackson.annotation.JsonTypeName;
+import no.unit.nva.commons.json.JsonSerializable;
+import nva.commons.core.JacocoGenerated;
+
+import java.net.URI;
+
+@JsonTypeInfo(use = JsonTypeInfo.Id.NAME, property = "type")
+@JsonTypeName("ChannelClaim")
+public record ChannelClaimDto(URI channelId, ChannelConstraintDto constraint) implements JsonSerializable {
```

Comment:
Why "channelId"? I think the "channel" was better as this is what the data represents, not "what it is" -> in our scheme "id" means "uri", so here you're saying channelUri, and the type is already clear.

Reply:
Changed back to channel :)